app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,5 +1,5 @@
 $primary: #1a454a;
```

Comment:
Is using the `HEX` color code here essential?
Could it be changed to an `RGB` code for consistency reasons?

Reply:
Isn't HEX default for HTML?
Agreed for the consistency.

Reply:
Done